crates/nu-utils/src/default_files/sample_config.nu

```diff
@@ -547,17 +549,18 @@ $env.config.color_config = (dark-theme)
 #     - 'i': italics
 #     - 'd': dimmed
 
-# foreground, background, and cursor colors are not handled by Nushell, but can be used by 
+# foreground, background, and cursor colors are not handled by Nushell, but can be used by
```

Comment:
I am unsure what to do here, example configuration should work out of the box, or else it is really confusing to try different options